From cf90a2ee429bf281273499993c18d9ebf0e4328c Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Thu, 27 Sep 2012 15:49:08 +0800
Subject: [PATCH 2094/2463] ENGR00226095-2 SPDIF: add HDMI AUDIO in device
 name in mx6_sl evk

because HDMI AUDIO in mx6sl evk comes from SPDIF TX, change to
HDMI AUDIO in device name.

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/imx/imx-spdif.c |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/sound/soc/imx/imx-spdif.c b/sound/soc/imx/imx-spdif.c
index efee769..13c97ae 100644
--- a/sound/soc/imx/imx-spdif.c
+++ b/sound/soc/imx/imx-spdif.c
@@ -1,7 +1,7 @@
 /*
  * ASoC S/PDIF driver for IMX development boards
  *
- * Copyright (C) 2008-2011 Freescale Semiconductor, Inc.
+ * Copyright (C) 2008-2012 Freescale Semiconductor, Inc.
  *
  * based on stmp3780_devb_spdif.c
  *
@@ -119,6 +119,11 @@ static int __init imx_spdif_init(void)
 		return -ENOMEM;
 	}
 
+	if (machine_is_mx6sl_evk()) {
+		imx_spdif_dai_link.name = "HDMI-Audio";
+		imx_spdif_dai_link.stream_name = "HDMI-Audio";
+	}
+
 	platform_set_drvdata(imx_spdif_snd_device, &snd_soc_card_imx_spdif);
 
 	ret = platform_device_add(imx_spdif_snd_device);
-- 
1.7.7.4

